back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/exporters/idea_pdf_form_exporter.rb

```diff
@@ -7,6 +7,7 @@ class IdeaPdfFormExporter < BaseFormExporter
     attr_reader :participation_context, :form_fields, :previous_cursor
 
     FORBIDDEN_HTML_TAGS_REGEX = %r{</?(div|span|ul|ol|li|img|a){1}[^>]*/?>}
+    JUMBLING_FIELD_TYPES = %w[multiline_text html_multiloc text text_multiloc]
```

Comment:
Wonder if this might make more sense in the TextCorrector itself

Reply:
Good question. I put it here because this class decides whether to use lines or not in `render_field -> render_text_lines`.